internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr_test.go

```diff
@@ -195,6 +189,95 @@ func TestKafkaManager_Reconcile(t *testing.T) {
+		accessControlListConfig *acl.AccessControlListConfig
+		kafkaConfig             config.KafkaConfig
+		billingModel            amsv1.BillingModel
+	}
+	tests := []struct {
+		name   string
+		fields fields
+	}{
+		{
+			name: "should value actual and desired kafka billing model",
+			fields: fields{
+				billingModel: "standard",
+				kafkaService: &services.KafkaServiceMock{
+					IsQuotaEntitlementActiveFunc: func(kafkaRequest *dbapi.KafkaRequest) (bool, error) {
+						return true, nil
+					},
+					DeprovisionExpiredKafkasFunc: func() *errors.ServiceError {
+						return nil
+					},
+					CountByStatusFunc: func(status []constants.KafkaStatus) ([]services.KafkaStatusCount, error) {
+						return []services.KafkaStatusCount{}, nil
+					},
+					ListAllFunc: func() (dbapi.KafkaList, *errors.ServiceError) {
+						return dbapi.KafkaList{
+							NewKafkaRequest(WithDefaultTestValues()),
+							NewKafkaRequest(WithDefaultTestValues(), WithActualBillingModel(""), WithDesiredBillingModel("")),
+							NewKafkaRequest(WithDefaultTestValues(), WithActualBillingModel(""), WithDesiredBillingModel("")),
+							NewKafkaRequest(WithDefaultTestValues(), WithActualBillingModel("marketplace"), WithDesiredBillingModel("marketplace")),
+							NewKafkaRequest(WithDefaultTestValues(), WithActualBillingModel(""), WithDesiredBillingModel("")),
+						}, nil
+					},
+					UpdateFunc: func(kafkaRequest *dbapi.KafkaRequest) *errors.ServiceError {
+						return nil
+					},
+				},
+				clusterService: &services.ClusterServiceMock{
+					FindStreamingUnitCountByClusterAndInstanceTypeFunc: func() (services.KafkaStreamingUnitCountPerClusterList, error) {
+						return services.KafkaStreamingUnitCountPerClusterList{}, nil
+					},
+				},
+				accessControlListConfig: acl.NewAccessControlListConfig(),
+				kafkaConfig: func() config.KafkaConfig {
+					cfg := config.NewKafkaConfig()
+					_ = cfg.ReadFiles()
+					return *cfg
+				}(),
+			},
+		},
+	}
+
+	for _, testcase := range tests {
+		tt := testcase
+
+		t.Run(tt.name, func(t *testing.T) {
+			g := gomega.NewWithT(t)
+			k := &KafkaManager{
+				kafkaService:            tt.fields.kafkaService,
+				clusterService:          tt.fields.clusterService,
+				dataplaneClusterConfig:  &tt.fields.dataplaneClusterConfig,
+				accessControlListConfig: tt.fields.accessControlListConfig,
+				kafkaConfig:             &tt.fields.kafkaConfig,
+				quotaServiceFactory: &services.QuotaServiceFactoryMock{
+					GetQuotaServiceFunc: func(quotaType api.QuotaType) (services.QuotaService, *errors.ServiceError) {
+						return &quota.AMSQuotaServiceMock{
+							GetSubscriptionByIDFunc: func(subscriptionID string) (*amsv1.Subscription, bool, *errors.ServiceError) {
+								subscription, _ := amsv1.NewSubscription().ClusterBillingModel(tt.fields.billingModel).Build()
+								return subscription, true, nil
+							},
+						}, nil
+					},
+				},
+			}
+
+			k.Reconcile()
```

Comment:
Here Reconcile is being called. Is it possible to just call the function being tested?